README.md

```diff
@@ -58,6 +58,44 @@ spec:
             - Resource
 ```
 
+
+When adding a new policy, please follow these guidelines:
+
+1. Create a new folder with the name of the policy.
+2. Add the Kyverno policy YAML file to the folder.
+3. Add an `artifacthub-pkg.yml` metadata file to the folder. See the [Template Policy Metadata File](https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml) section for an example and instructions on how to complete the contents of this file.
```

Comment:
Change second sentence to:

"See an example metadata file for Kyverno policies below and customize per the comments."